active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,419 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(model_accuacy_arr)\n",
+    "plt.xticks(range(num_rounds))\n",
+    "plt.xlabel(\"Round\")\n",
+    "plt.ylabel(\"Model Accuracy\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the plot above, we can see that the model accuracy increases steadily with each additional round of collecting more labels, getting improved consensus labels, and model training."
+   ]
+  }
```

Comment:
also show how to get annotator quality scores at the end of the notebook?

Reply:
Added a section at the end of notebook that demonstrates basic crowdlab functionality!